app/assets/html/sidebar.html

```diff
@@ -18,7 +18,7 @@
                 <div class="conversation-preview menu-item subitem" ng-repeat="conversation in conversationsByUserId[gladiatorId]" ng-show="arenaStateByUserId[gladiatorId]" ng-click="conversationPreviewClicked(conversation.id)">
                     <div class="conversation-heading">
                         <div class="conversation-actions">
-                            <div class="conversation-advance glyphicon glyphicon-arrow-right btn" ng-click="conversationPreviewClicked(conversation.id)"></div>
+                            <div class="conversation-advance glyphicon glyphicon-remove btn" ng-click="deleteIconClicked(conversation.id); $event.stopPropagation();"></div>
```

Comment:
1. Yea, we should maybe show a warning message? Like "Are you sure you want to delete this conversation? This cannot be undone"